Add trailing slash rewrite rule to the "Rewrites on Cloud" article (#2894)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Manage-Hostnames/Rewrites-on-Cloud/index.md

:::note
This redirect rule will not apply when the request is already going to the secure HTTPS URL. This redirect rule will also not apply on your local copy of the site running on `localhost`.
:::

## Adding a trailing slash to your URLs

It is possible to transform all of your URLs to use a trailing slash consistently for SEO.

> To accomplish this, add a rewrite rule to the live environment's `web.config` in the `<system.webServer><rewrite><rules>` section.

Reviewer: ```suggestion
To accomplish this, add a rewrite rule to the Live environment's `web.config` in the `<system.webServer><rewrite><rules>` section.
```

> For example, the following rule will redirect all requests for the site https://mysite.com/page URL to the page https://mysite.com/page/ URL and respond with a permanent redirect status. This way you can ensure that you use the trailing slashes consistently on your site.

Reviewer: ```suggestion
For example, the following rule will redirect all requests for `https://mysite.com/page` to `https://mysite.com/page/`, and respond with a permanent redirect status. This way you can ensure that you use the trailing slashes consistently on your site.
```

```xml
<rule name="Add trailing slash" stopProcessing="true">
  <match url="(.*[^/])$" />
  <conditions>
    <add input="{REQUEST_FILENAME}" matchType="IsDirectory" negate="true" />
    <add input="{REQUEST_FILENAME}" matchType="IsFile" negate="true" />
    <add input="{REQUEST_URI}" pattern="^/media" negate="true" />
    <add input="{REQUEST_URI}" pattern="^/umbraco" negate="true" />
    <add input="{REQUEST_URI}" pattern="^/DependencyHandler.axd" negate="true" />
    <add input="{REQUEST_URI}" pattern="^/App_Plugins" negate="true" />
    <add input="{REQUEST_URI}" pattern="^/\.well-known/acme-challenge" negate="true" />
  </conditions>
  <action type="Redirect" url="{R:1}/" />
</rule>
```

:::note

> Take note of the negates in the rewrite rule. Especially the negate for the media folder is important because your media will not show correctly after [your site has been migrated to use Azure Blob Storage](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Media/)

Reviewer: Based on your reply, I've tried to rewrite this a tiny bit, so make a little clearer! What do you think?
Also, should we change this note to a warning instead, and place it above the rewrite snippet? 🤔 Seems like something people should be made aware of BEFORE they add the snippet to their code.

What do you think?

```suggestion
Take note of the negates in the rewrite rule. 

It is especially important to negate the path to the media folder because with the trailing slash added, your media will not show correctly after [your site has been migrated to use Azure Blob Storage](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Media/).
```
